app/manifest/v2/chrome.json

```diff
@@ -1,5 +1,5 @@
 {
-  "content_security_policy": "frame-ancestors 'none'; script-src 'self' 'wasm-unsafe-eval'; object-src 'self'",
+  "content_security_policy": "frame-ancestors 'none'; script-src 'self' 'wasm-unsafe-eval'; object-src 'none'",
```

Comment:
I'm not aware of anyplace we're using `<object>` or `<embed>` elements, so this is probably a safe change. I was the one who set `object-src` to `self` in the first place. My motivation for doing that was just that it matched the browser default CSP.